tools/sim/bridge.py

```diff
@@ -82,9 +83,13 @@ def imu_callback(imu, vehicle_state):
   dat.sensorEvents[1].gyroUncalibrated.v = [imu.gyroscope.x, imu.gyroscope.y, imu.gyroscope.z]
   pm.send('sensorEvents', dat)
 
-def panda_state_function():
+def panda_state_function(q: Queue):
   pm = messaging.PubMaster(['pandaState'])
   while 1:
+    if not q.empty():
+      if q.get() == "quit":
+        break
```

Comment:
I think a threading.Event is much cleaner, for example this becomes:
```suggestion
  while not exit_event.is_set():
```

Reply:
Thanks; done! I've renamed some things as a result of this change, too.